tests/test_search.py

```diff
@@ -52,3 +52,23 @@ def test_objects_are_escaped(app, status, warning):
 
     index = jsdump.loads(searchindex[16:-2])
     assert 'n::Array&lt;T, d&gt;' in index.get('objects').get('')  # n::Array<T,d> is escaped
+
+def assert_lang_agnostic_key_words(searchindex):
+    assert 'findnotthiskey' not in searchindex
+    assert 'thisnoteith' not in searchindex
+    assert 'thistoo' in searchindex
+    assert 'thisonetoo' in searchindex
+
+@with_app(testroot='search')
+def test_meta_keys_are_handled_for_language_en(app, status, warning):
+    app.builder.build_all()
+    searchindex = (app.outdir / 'searchindex.js').text()
+    assert_lang_agnostic_key_words(searchindex)
+    assert 'findthiskei' in searchindex
+
+@with_app(testroot='search-de')
```

Comment:
We can override confvals by `confoverrides` option of `with_app` decorator.
Could you replace this line with `@with_app(testroot='search', confoverrides={'html_search_language'='de'})`?
Then `roots/test-search-de` becomes needless and it keeps testroots simple.